modules/client-configuration/pages/cobbler.adoc

```diff
@@ -53,12 +54,9 @@ A sync is basically a rebuild of every file cobbler touched. I will try give you
 To use Cobbler for system installation with PXE, you will require a TFTP server.
 {productname} installs a TFTP server by default.
 To PXE boot systems, you will require a DHCP server, or have access to a network DHCP server.
+// ke, 2019-11-26: Maybe, we should say that this on SUMA has to be the address of the SUMA server?
 Edit the [path]``/etc/dhcp.conf`` configuration file to change [option]``next-server`` to the hostname or IP address of your Cobbler server.
```

Comment:
The DHCP Server is not a feature of cobbler nor it is provided by SUSE Manager. So we talk about any dhcp server in the company network. It can be dhcpd from a SLES or RedHat, but it can also be dnsmasq or a Microsoft DHCP server.

Describing "how to configure it" might be difficult. We can say "If your dhcp server is a XXXXX you configure it this way" but if you use something else, you need to "adapt" this description.

cobbler has the feature to use its template mechanism to configure dhcp, but this is **not supported** with SUSE Manager.

Reply:
Ok, then I'd better just remove line 58 (and 57).